### PR TITLE
Prac 08 code review request

### PR DESCRIPTION
**Pull request by DanL-Smith**
Hello @CallumGracie95, Please check formatting, naming and logic.

### CODE REVIEW

File: prac_08/silver_service_taxi.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+Taxi class
+"""
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Represent a SilverServiceTaxi."""
+    FLAGFALL = 4.50
```
Comment (lindsaymarkward) [inline prac_08/silver_service_taxi.py:10]: This should not be a constant. We might want to change it.

File: prac_08/silver_service_taxi_test.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    """Test the Silver Service class."""
+    my_silver_service_taxi = SilverServiceTaxi("First Class Taxi", 100, 2)
+    my_silver_service_taxi.start_fare()
+    my_silver_service_taxi.drive(18)
+    print(my_silver_service_taxi)
+    print(F"Current fare: ${my_silver_service_taxi.get_fare()}")
```
Comment (lindsaymarkward) [inline prac_08/silver_service_taxi_test.py:15]: How do you know if this worked correctly?

File: prac_08/taxi.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+Taxi class
+"""
+from prac_08.car import Car
+
+
+class Taxi(Car):
+    """Specialised version of a Car that includes fare costs."""
+    PRICE_PER_KM = 1.23
```
Comment (lindsaymarkward) [inline prac_08/taxi.py:10]: Also not a constant

File: prac_08/taxi_simulator.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            while number < 0:
+                print('Number must be 0 or greater')
+                number = int(input(prompt))
+        except ValueError:
+            print("You must enter a number")
+            continue
+        return number
+
+
+def determine_current_taxi(current_taxi, taxis: list):
```
Comment (lindsaymarkward) [inline prac_08/taxi_simulator.py:68]: determine is an odd name. It looks like it "gets" a taxi.

File: prac_08/taxi_test.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+def main():
+    """Test the Taxi class"""
+    my_taxi = Taxi("Prius 1", 100)
+    my_taxi.drive(40)
+    print(my_taxi)
+    my_taxi.start_fare()
+    my_taxi.drive(100)
+    print(my_taxi)
+    print(F"Current fare: ${my_taxi.get_fare()}")
```
Comment (lindsaymarkward) [inline prac_08/taxi_test.py:18]: How do you know if this worked correctly?

File: prac_08/unreliable_car_test.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def main():
+    """Test the Unrealiable Car class."""
+    my_unreliable_car = UnreliableCar("Beat-up Holden", 100, 77)
+    my_unreliable_car.drive(40)
+    print(my_unreliable_car)
+    print(my_unreliable_car.reliability)
+    my_unreliable_car.drive(100)
+    print(my_unreliable_car)
```
Comment (lindsaymarkward) [inline prac_08/unreliable_car_test.py:16]: Only two tests on only one car is not enough when random is involved

File: prac_08/silver_service_taxi.py (hunk 1 above)
Comment (DanL-Smith) [reply prac_08/silver_service_taxi.py:10]: I have now changed this. Thank you.

File: prac_08/taxi.py (hunk 3 above)
Comment (DanL-Smith) [reply prac_08/taxi.py:10]: Thanks for the feedback.

File: prac_08/taxi_simulator.py (hunk 4 above)
Comment (DanL-Smith) [reply prac_08/taxi_simulator.py:68]: You're right, "get" makes much more sense here. :)

**Pull request merged**